### ISSUE TITLE
Unload chunks that haven't been needed for some time

### ISSUE THREAD

**Issue by rom1504** [PrismarineJS/prismarine-world#20]
it's now possible since we have chunk loading/unloading.

One way to do it would be using the timestamp stored in region files

**Comment by rom1504** [comment]
https://en.wikipedia.org/wiki/Least_frequently_used

**Comment by Karang** [comment]
https://en.wikipedia.org/wiki/Cache_replacement_policies#LRU would be a better policy IMO

**Comment by rom1504** [comment]
Yeah I was thinking about the case of a server where the spawn point might be unloaded with LRU if everyone disconnects and someone go explore somewhere far.
But LFU has its own issues.

LRU seems like a simple policy that should work, especially if the cache is big enough (like 20x20 chunks)
